README.md

```diff
@@ -60,6 +60,11 @@ as long as they're present in the channel mapping.
     "ircOptions": { // Optional node-irc options
       "floodProtection": false, // On by default
       "floodProtectionDelay": 1000 // 500 by default
+    },
+    "slackOptions": { // Optional node-irc options for Slack
```

Comment:
I concede that ircOptions is intended to contain all options passed to node-irc; however, some options are for Slack -> IRC behavior and some are for IRC -> Slack behavior.  Creating this distinction in the settings will aid users in properly configuring their bot. If it gives everyone a warm and fuzzy feeling to have only one variable sent to node-irc, we can create an 'Options' parent with ircOptions and slackOptions as children objects.  Pretty easy to do in JSON.  If I have time today, I'll code it up.